### ISSUE TITLE
Bulk Update - UI Changes

### ISSUE THREAD

**Issue by LynnMcRae** [sul-dlss/argo#87]
This is a simple rearranging of the current elements for clarity and space since it confuses some users that new questions open up below the bottom of the screen.  There are placeholders for new features which can be ignored. Image can be rev'ed to match current feature set.

![bulk-update](https://cloud.githubusercontent.com/assets/1824732/10872168/80544b9a-80b0-11e5-9dd0-28736e8e2d0e.jpg)

**Comment by LynnMcRae** [comment]
Revised picture showing design applied to current set of buttons minus what is being dropped. 
![bulk-update2](https://cloud.githubusercontent.com/assets/1824732/11546197/984e070a-9900-11e5-9886-117bce22928f.png)

**Comment by tingulfsen** [comment]
What is the significance of the green numbers 1, 2, 3 next to the three main columns?

**Comment by blalbrit** [comment]
I think @LynnMcRae was trying to indicate a step-wise progression.

**Comment by mejackreed** [comment]
I thought we discussed any changes here not to be considered feature parity?

**Comment by blalbrit** [comment]
Agreed. We started with just "eliminate unused buttons". I might have missed the bit where @LynnMcRae specified new layout - but maybe he can weigh in. First priority would be to stick with UI changes to this page being enhancements, not parity.

**Comment by LynnMcRae** [comment]
Eliminating buttons means moving what's left around, so we've breached the don't touch the page barrier. The new layout is just the same stuff in more rational places taking less space to at least eliminate the final step always occurring out of sight beneath the fold. No functional or behavioral changes. Tommy said he'd do just the ez non-threatening parts.

**Comment by LynnMcRae** [comment]
Ben said "I think @LynnMcRae was trying to indicate a step-wise progression." ... yes, we got feedback that folks didn't connect "Versioning" as a grouping on the far left with "you must prepare first" on the far right, and we neglected to include the same warning with the "Metadata" category in the middle.  The page layout is a mess on its own and leads to errors.

**Comment by tingulfsen** [comment]
Do we really intend to have 2 "Purge" buttons on this page, @LynnMcRae ?

**Comment by LynnMcRae** [comment]
d'oh!  the first Purge is the misplaced one, given you are about to blast the object away.

**Comment by LynnMcRae** [comment]
Here's the revised mockup we discussed earlier today. It fixes the druid gathering interaction, consolidates three successively reveals boxes into one shown from the starts, and simplified the final interaction for consistency.
![bulk-update2](https://cloud.githubusercontent.com/assets/1824732/12183705/5593e45a-b545-11e5-865c-89262ff53073.png)

As we discussed, some deltas are fine, e.g., if it's not trivial to have same-sized buttons, then just left justify each set.  Note one layout change; we didn't see why "Add a workflow" was in the first group so we moved it to the second, leaving it red because it's use is tricky and needs some caution.
